src/Image.php

```diff
@@ -284,20 +284,30 @@ public function listOperations()
      * @param array  $stackOperations Stack operations
      * @param string $organization    Optional organization name
      * @param array  $stackOptions    Stack options
+     * @param bool   $overwrite       If an existing stack should be overwritten
      *
      * @return Stack
      */
-    public function createStack($stackName, array $stackOperations, $organization = '', array $stackOptions = [])
+    public function createStack(
+                                $stackName,
+                                array $stackOperations,
+                                $organization = '',
+                                array $stackOptions = [],
+                                $overwrite = false
+    )
     {
         $stackData = [
             'operations' => $stackOperations,
             'options' => $stackOptions,
         ];
-
+        $queryString = '';
+        if ($overwrite) {
+            $queryString = '?overwrite=true';
+        }
         $contents = $this
             ->call(
                 'PUT',
-                implode('/', [self::STACK_RESOURCE, $this->getOrganization($organization), $stackName]),
+                implode('/', [self::STACK_RESOURCE, $this->getOrganization($organization), $stackName]) . $queryString,
```

Comment:
I'd recommend to use the `query` request option instead, i.e. ['json' => $stackData, 'query' => ['overwrite' => 'true']]`